scripts/main/view.js

```diff
@@ -493,13 +493,13 @@ view.album = {
 	},
 
 	public: function () {
-		$("#button_visibility_album").removeClass("active--not-hidden active--hidden");
```

Comment:
I don't think the changes to this file are needed at all. What this code does is it changes the _color_ of the "eye" button, depending on the public/hidden settings. See here:

https://github.com/bastantoine/Lychee-front/blob/d59325b793b834337a94f217b549a8ef6d628985/styles/main/_header.scss#L114-L120

But in case of your button, which is always gray, this won't do anything...